.github/workflows/ci.yml

```diff
@@ -25,9 +25,21 @@ jobs:
           mix local.rebar --force
           mix local.hex --force
           mix deps.get
-      - name: Compile Code
-        run: mix compile --warnings-as-errors
       - name: Ensure Code is Formatted
         run: mix format --check-formatted
+      - name: Compile for Tests
+        run: MIX_ENV=test mix compile --warnings-as-errors
       - name: Run Tests
         run: mix test
+      - name: Generate Release
+        if: github.ref == 'refs/heads/master'
+        run: mix release && cp _build/prod/rel/bakeware/fika fika
```

Comment:
The `cp` here was added to ensure that we have control over the resulting binary's name